terraform.tfvars.example

```diff
@@ -102,4 +106,5 @@ attached_drg_id = null
 #subnets = {
 #  sub1 = {name = "subnet1",cidr_block = "10.0.4.0/24"}
 #  sub2 = {cidr_block="10.0.5.0/24",type="private"}
```

Comment:
your default in the code seems to be public if type is not specified? I think the default should be private if not specified.

Reply:
@hyder  - This change is not part of this PR. Please note . If needed a separate issue need to be opened to fix this